models/report.py

```diff
@@ -311,7 +319,7 @@ def scan_datastore(self, ds, process_step, session=None):
             if self.warnings is None or self.warnings == '':
                 self.warnings = new_string
             else:
-                self.warnings += ', ' + new_string
+                self.warnings += '\n***|***|***\n' + new_string
```

Comment:
Now I understand what you meant in that comment; I was confused at first, because I thought `"***"` meant a warning, not literal `"***"`.  I was confused as to why three warnings separated by |, then a newline.  This here makes much more sense.

Reply:
Yeah I didn't think I made it very clear. But I think this will be good.